CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # Changelog
 
+## 0.10.0
+
+* `nerves_hub_link_common 0.2.0`
+
+* Enhancements
+  * Report `device_api_version` in web connections
+  * Report `currently_downloading_uuid` when joining channel to aid
+    in resumable downloads
+  * Optional `NervesHubLink.Client.reboot/0` callback behavior
```

Comment:
How about
Perhaps this?

```suggestion
* Enhancements
  * Report API version to NervesHub. While there's currently only one version of the Device API, this is anticipation that we may want to change it in the future.
  * On reconnect, notify NervesHub of firmware currently being downloaded so that NervesHub can differentiate failed firmware updates from network hiccups.
  * Allow devices to implement their own reboot logic by implementing the `NervesHubLink.Client.reboot/0` callback
```